r/tools/nixlibs.R

```diff
@@ -116,33 +105,56 @@ download_binary <- function(lib) {
   # validate binary checksum for CRAN release only
   if (!skip_checksum && dir.exists(checksum_path) && is_release ||
     enforce_checksum) {
```

Comment:
With this logic now we will skip the checksum (in this case, get to this function and then reply with `checksum_ok == TRUE` if any of these are true:

* the skip var is set
* it's not a release
* the directory `tools/checksums` does not exist

Is that last one right? I get that we're trying to be protective here, and if there's no checksum file we will fail the checksum. But do we want to have this particular escape hatch here rather than failing the checksum (and then presumably building from source)?

Thoughts @assignUser ?